### PR TITLE
add tag service on some metrics

### PR DESCRIPTION
**Pull request by dbenque**
Add tag `service` to some of the metrics.
The tag `service` will allow us to connect to service catalog for some analysis (who is associated with the service) but also will ease future integration with SDP (push some NLA relevant information to SDP).

### CODE REVIEW

File: cmd/draino/draino.go [hunk 1]
```diff
@@ -65,101 +65,6 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
-type filtersDefinitions struct {
```
Comment (dbenque) [inline cmd/draino/draino.go:68]: This code relative to filter initialisation was moved into `internal/kubernetes/config`. We have now a dedicated structure for pod and node filtering function that we can pass to other building blocks.

File: internal/kubernetes/config.go [hunk 2]
```diff
@@ -15,3 +21,114 @@ type GlobalConfig struct {
...
+	// CandidatePodFilter, Should the pod block the node for being candidate ?
+	CandidatePodFilter PodFilterFunc
+	// DrainPodFilter, Should the pod be drained ?
+	DrainPodFilter PodFilterFunc
+
+	// NodeLabelFilter, Is the node eligible (label checks only) ?
+	NodeLabelFilter NodeLabelFilterFunc
+}
+
+func GenerateFilters(cs *kubernetes.Clientset, store RuntimeObjectStore, log *zap.Logger, options FilterOptions) (FiltersDefinitions, error) {
```
Comment (dbenque) [inline internal/kubernetes/config.go:49]: code copied from main.go, no change in the logic.

File: cmd/draino/draino.go [hunk 3]
```diff
@@ -247,7 +152,21 @@ func main() {
 			NodesStore:                 nodes,
 		}
 
-		filtersDef, err := generateFilters(cs, store, zlog, options)
+		filteringOptions := kubernetes.FilterOptions{
```
Comment (adrienjt) [inline cmd/draino/draino.go:155]: Why not bind the flags directly to a kubernetes.FilterOptions?

**Review by adrienjt — APPROVED** [review]

File: cmd/draino/draino.go (hunk 3 above)
Comment (dbenque) [reply cmd/draino/draino.go:155]: options fields have gone through Validation that could have been defaulted/mutated compare to flags.

**Pull request merged**